### PR TITLE
Add support for dark mode

### PR DESCRIPTION
**Pull request by mohamedelhefni**
this PR is addressing the feature requested at #606 

dark

### Changes
* Migrate to bootstrap v5.3
* Remove custom css that forces light theme 
* add toggle theme functionality 
  * change editor theme 
  * change the `data-bs-theme` attribute

### CODE REVIEW

**Comment by sosedoff** [comment]
Can you break out bootstrap upgrade into a separate PR? That'll be something we would need to merge first and is definitely easier to check/verify independently from the other changes you're proposing.

File: static/index.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+
 <head>
   <title>pgweb</title>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta http-equiv="Content-Language" content="en">
-  <link rel="stylesheet" href="static/css/bootstrap.css"></link>
-  <link rel="stylesheet" href="static/css/font-awesome.css"></link>
-  <link rel="stylesheet" href="static/css/app.css"></link>
+  <link rel="stylesheet" href="static/css/bootstrap.min.css">
```
Comment (sosedoff) [inline static/index.html:9]: Keep the old formatting here

File: static/index.html [hunk 2]
```diff
@@ -35,14 +40,17 @@
       </ul>
 
       <div class="connection-actions">
-        <a href="#" id="edit_connection" class="btn btn-default btn-sm"><i class="fa fa-database"></i> Connect</a>
-        <a href="#" id="close_connection" class="btn btn-default btn-sm">Disconnect</a>
+        <a href="#" id="edit_connection" class="btn btn-default text-white btn-sm"><i
```
Comment (sosedoff) [inline static/index.html:43]: Are you using some kind of auto formatter for html here? The tag content looks broken over the lines

File: static/js/app.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
-var appInfo             = {};
-var appFeatures         = {};
-var editor              = null;
-var connected           = false;
-var bookmarks           = {};
-var default_rows_limit  = 100;
-var currentObject       = null;
+var appInfo = {};
```
Comment (sosedoff) [inline static/js/app.js:1]: Please keep the original formatting in this file

**Comment by mohamedelhefni** [comment]
Hi @sosedoff 
I have a strong interest in the pgweb project.  
Initially, I noticed the absence of vim mode keybindings, which prompted me to contribute to the project. While working on it,
I identified an issue related to adding support for a dark theme. Initially, I thought upgrading Bootstrap to v5.3 would seamlessly enable dark theme support, but this process ended up causing complications.
The upgrade adversely affected elements like the context menu and dropdown menu, and since the context menu plugins are no longer actively maintained, resolving these issues would take more time than anticipated.
Regrettably, I have decided to close this pull request.

**Pull request closed without merging**